docs/src/mdx-components.tsx

```diff
@@ -162,6 +183,9 @@ export const a = function Anchor({
     </span>
   );
 };
+
 export const Callout = ({ icon = true, ...props }) => (
   <Alert icon={icon} spaceAfter="large" {...props} />
 );
+
+export const Examples = ({ exampleId }) => <Example exampleId={exampleId} />;
```

Comment:
Given that the prop is exactly the same, I don't see a benefit of including it in this file. It can be included like the other components that are just copies in DocLayout. Then you don't have to get into the confusion of having different naming (`Examples`).